recipe/build-sysroot.sh

```diff
@@ -33,4 +33,8 @@ ln -s $PWD/usr/lib64 $PWD/usr/lib
 cp "${SRC_DIR}"/binary-freebl/lib64/libfreebl3.so ${PWD}/lib64/.
 ln -s ${PWD}/lib64/libfreebl3.so ${PWD}/usr/lib64/libfreebl3.so
 
+sed -i.bak 's/AS_NEEDED/AS_NEEDED ( /lib64/librt.so.1 ) AS_NEEDED/g' usr/lib64/libc.so
```

Comment:
```suggestion
sed -i.bak 's,AS_NEEDED,AS_NEEDED ( /lib64/librt.so.1 ) AS_NEEDED,g' usr/lib64/libc.so
```

using commas instead of `/` helps sed. you can use any character as separator